### PR TITLE
Otp 309 add lcp

### PR DESCRIPTION
**Pull request by yaniv-li**
https://cloudinary.atlassian.net/browse/OTP-319

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ const conf = {
       "location": process.env.WTP_LOCATION_PATH || 'data.location',
       "url": process.env.WTP_URL_PATH || 'data.url',
       "browserName": process.env.WTP_BROWSER_NAME_PATH || 'data.median.firstView.browser_name',
-      "browserVer": process.env.WTP_BROWSER_VER_PATH || 'data.median.firstView.browser_version'
+      "browserVer": process.env.WTP_BROWSER_VER_PATH || 'data.median.firstView.browser_version',
+      "lcp": process.env.LCP_PATH || "data.median.firstView.largestPaints",
+      "lcpURL": process.env.LCP_PATH || "data.median.firstView.LargestContentfulPaintImageURL"
```
Comment (mikeys) [inline config/default.js:36]: You probably want to modify the environment variable name to `LCP_URL_PATH` as currently it's the same as the `lcp` one.

File: wtp/wtpResultsParser.js [hunk 2]
```diff
@@ -95,6 +99,17 @@ const parseTestResponse = (body, rollBarMsg) => {
   return body.data.testId;
 };
 
+const extractLCP = (paintsArray) => {
+  let imagePaints = paintsArray.filter((p) => {
+    return p.event === 'LargestContentfulPaint' && p.type === 'image';
+  });
+  if (imagePaints.length === 0) {
```
Comment (mikeys) [inline wtp/wtpResultsParser.js:106]: @yaniv-li I saw cases where there were multiple `LargestImagePaint` events (like in [this](https://www.webpagetest.org/jsonResult.php?test=210613_AiDcKE_9fc00c50ba5bb22a229f191d3ee4217f&pretty=1) case). Not exactly sure why. In that case, perhaps worth taking the largest one ?

Comment (yaniv-li) [reply wtp/wtpResultsParser.js:106]: @mikeys this is why I filter with the type too the other one is text content  if nothing is found (this is what happens if you test a file in github) I go for  "LargestImagePaint" entry

Comment (mikeys) [reply wtp/wtpResultsParser.js:106]: I'm saying that there are two `LargestImagePaint` events... one with `time: 441` and another with `time: 976`.

Comment (yaniv-li) [reply wtp/wtpResultsParser.js:106]: added sort by time

**Review by mikeys — APPROVED** [review]

**Pull request merged**